apps/examples/lcd_test/example_lcd.c

```diff
@@ -387,7 +439,11 @@ int lcd_test_main(int argc, char *argv[])
 		count++;
 		printf("count :%d\n", count);
 	}
+	touch_test();
```

Comment:
No need to call this , it is already called in task create

Reply:
done